pretrain/validation.py

```diff
@@ -144,7 +144,11 @@ def check_for_reasonable_output(
 
 
 def compute_losses(
-    model, batches: typing.List[torch.Tensor], device: str, pad_token_id: int
+        model,
+        batches: typing.List[torch.Tensor],
+        device: str,
+        pad_token_id: int,
+        pack_samples: bool,
```

Comment:
This is not a bool to say if the samples should be packed, but saying if they were right?

Perhaps change parameter name to better indicate that. "packed_samples" or "sample_packing_used"

Reply:
that would be more accurate. I agree.
Done